### PR TITLE
Add concurrency

### PR DESCRIPTION
**Pull request by joostverdoorn**
This PR adds the option of concurrency to gennie. When set to 1, gennie behaves exactly as it used to. 

This delivers a significant performance increase when I test-run it, making all my cores work instead of just one.

@inventid/tech ready for review!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,16 +12,16 @@ var Nightmare  = require('nightmare'),
 // bluebird's promises.
 Queue.configure(Promise);
 
-var nightmare = Nightmare(),
-    maxConcurrent = 1,
-    maxQueue = Infinity,
-    queue = new Queue(maxConcurrent, maxQueue);
+var concurrency = config.concurrency,
```
Comment (rogierslag) [inline index.js]: could you add `process.env.CONCURRENCY || config.concurrency`? That way this is easier once the thing is in Docker (you do not need to edit config etc)

**Comment by rogierslag** [comment]
One small change which would make my life easier, but otherwise :+1: 

Once this is merged, can you release it? (That will also trigger auto docker builds etc)

File: config.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "port": 3000,
-  "document_path": "./document",
-  "render_timeout": 30000
+  "concurrency": 4,
+  "document_path": "../gennie-documents/eticket",
```
Comment (rogierslag) [inline config.json]: this breaks the docker image, can you set it back?

Comment (joostverdoorn) [inline config.json]: whoops

**Comment by rogierslag** [comment]
:+1:

**Pull request merged**